### PR TITLE
package/py-wand_add_versions

### PR DESCRIPTION
**Pull request by Sinan81**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-wand/package.py [hunk 1]
```diff
@@ -14,7 +14,9 @@ class PyWand(PythonPackage):
 
     license("MIT")
 
+    version("0.6.13", sha256="f5013484eaf7a20eb22d1821aaefe60b50cc329722372b5f8565d46d4aaafcca"
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/py-wand/package.py]: Hmm, this looks like a syntax error.

**Review by alalazo — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-wand/package.py (hunk 1 above)
Comment (Sinan81) [reply var/spack/repos/builtin/packages/py-wand/package.py]: fixed.

**Comment by Sinan81** [comment]
bug fixed. please merge.

**Review by alalazo — APPROVED** [review]
I'll let @adamjstewart do a final review and merge the PR

**Pull request merged**